spec/components/npq_separation/admin/adjustments_table_component_spec.rb

```diff
@@ -1,14 +1,17 @@
 require "rails_helper"
 
 RSpec.describe NpqSeparation::Admin::AdjustmentsTableComponent, type: :component do
-  subject { render_inline described_class.new(adjustments: adjustments, show_total: show_total) }
+  include Rails.application.routes.url_helpers
```

Comment:
Rather than pull all the url_helpers (and there are many) into the spec class, how about

```ruby
let(:urls) { Rails.application.routes.url_helpers }
```

then just referencing `urls.some_path` lower down

Reply:
👍🏽  ah yes - I remember this from another PR 😅